client/post-editor/controller.js

```diff
@@ -107,11 +108,38 @@ function getPressThisContent( query ) {
 	return pieces.join( '\n\n' );
 }
 
+function startEditingPostCopy( siteId, postId ) {
+	wpcom.site( siteId ).post( postId ).get().then( post => {
+		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
+		actions.startEditingNew( siteId, {
+			type: 'post',
+			content: post.content,
+		} );
+
+		const postAttributes = pick(
+			post,
+			'canonical_image',
+			'excerpt',
+			'featured_image',
+			'format',
+			'metadata',
+			'post_thumbnail',
+			'title'
+		);
+		postAttributes.categories = map( post.categories, 'ID' );
+		postAttributes.tags = map( post.tags, 'name' );
+
+		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
+		actions.edit( postAttributes );
+	} );
+}
+
 module.exports = {
 
 	post: function( context ) {
 		const postType = determinePostType( context );
 		const postID = getPostID( context );
+		const copyId = context.query.copy || false;
```

Comment:
It's the template ID.
I'm going to replace it with a more explicit `const postToCopyId`.